reactor-core/src/main/java/reactor/core/publisher/Hooks.java

```diff
@@ -570,6 +571,31 @@ public static void resetOnNextError() {
 	Hooks() {
 	}
 
+	/**
+	 *
+	 * @deprecated Should only be used by the instrumentation, DOES NOT guarantee any compatibility
```

Comment:
nit: there seem to be a misalignment

Reply:
(same in javadoc below)

Reply:
Sorry, not sure I see it 😅 Which misalignment?

Reply:
nevermind, seems to be an artifact of github's rendering on my machine (verified there's no space vs tab or missing space issue locally) 👍